ipatests/test_integration/test_sssd.py

```diff
@@ -148,3 +149,36 @@ def test_is_user_filtered(self, user):
             dp_req = ("Looking up [{0}] in data provider".format(
                 self.users[user]['name']))
             assert not dp_req.encode() in sssd_log
+
+    def test_extdom_group(self):
+        """ipa-extdom-extop plugin should allow @ in group name.
+
+        Test for : https://bugzilla.redhat.com/show_bug.cgi?id=1746951
+        """
+        sssd_conf_backup = tasks.FileBackup(self.master, paths.SSSD_CONF)
+        try:
+            tasks.modify_sssd_conf(
+                self.master,
+                self.master.domain.name,
+                {
+                    're_expression': '((?P<name>.+)@(?P<domain>[^@]+$))'
+                },
+                provider='sssd'
+            )
+            tasks.clear_sssd_cache(self.master)
+            self.master.run_command(['cat', paths.SSSD_CONF])
```

Comment:
What is the purpose of `$cat` here? Make use of assert if you are interested in particular attribute from `sssd.conf`.